invenio_app_ils/ui/backoffice/src/pages/ItemsSearch/ItemsSearch.js

```diff
@@ -0,0 +1,176 @@
+import React, { Component } from 'react';
+import {
+  Container,
+  Grid,
+  Segment,
+  Icon,
+  Header,
+  Button,
+  Dropdown,
+} from 'semantic-ui-react';
+import {
+  ReactSearchKit,
+  SearchBar,
+  ResultsList,
+  ResultsLoader,
+  ResultsPerPage,
+  EmptyResults,
+  Error,
+  Pagination,
+  Count,
+} from 'react-searchkit';
+import { apiConfig } from 'common/api/base';
+import { URLS } from 'common/urls';
+import { Error as IlsError } from 'common/components';
+import { item as endpoint } from 'common/api/item';
+import {
+  SearchBar as ItemsSearchBar,
+  ResultsList as ItemsResultsList,
+} from './components';
+import './ItemsSearch.scss';
+
+const resultsPerPageValues = [
+  {
+    text: '10',
+    value: 10,
+  },
+  {
+    text: '20',
+    value: 20,
+  },
+  {
+    text: '50',
+    value: 50,
+  },
+];
+
+export class ItemsSearch extends Component {
+  _renderSearchBar = (_, queryString, onInputChange, executeSearch) => {
+    return (
+      <ItemsSearchBar
+        currentQueryString={queryString}
+        onInputChange={onInputChange}
+        executeSearch={executeSearch}
+      />
+    );
+  };
+
+  _renderResultsList = results => {
+    return (
+      <div className="results-list">
+        <ItemsResultsList
+          results={results}
+          viewDetailsClickHandler={itemPid =>
+            this.props.history.push(URLS.itemDetails(itemPid))
+          }
+        />
+      </div>
+    );
+  };
+
+  _renderEmptyResults = (queryString, resetQuery) => {
```

Comment:
is it possible to make it reusable for other searches? Because I can see that some bits are repeated in the LoansSearch

Reply:
Yep, but as discussed in the DevOps, it is a deliberate choice, I don't know how this thing will evolve and the look and feel is very undone now...
I have the feeling we will pass through the full UI when making it more sexy, and the we can do this refactorings.

Reply:
As discussed, we will refactor this in the future